### ISSUE TITLE
Numpy Traceback Issue

### ISSUE THREAD

**Issue by BradyAJohnston** [JacquesLucke/animation_nodes#1162]
I'm on the latest release candidate (Version 2.80 (2.80 2019-07-24)) and when I go to enable with 2.1.4 I get the same traceback issue with numpy failing. I see someone has the same issue on Windows that I'm getting on macOS.

I'm running the official rc3 from blender.org and have also tried on the cmake compiled version from https://git.blender.org/blender.git with no luck.

![image](https://user-images.githubusercontent.com/36021261/62032478-5405b780-b21c-11e9-8b94-dea3d23cbda7.png)

Might be more of a Blender issue because trying to import from python console in Blender I get this:

```python
>>> import numpy
Traceback (most recent call last):
  File "<blender_console>", line 1, in <module>
  File "/Users/brady/Library/Application Support/Blender/2.80/scripts/addons/numpy/__init__.py", line 189, in <module>
    from . import random
  File "/Users/brady/Library/Application Support/Blender/2.80/scripts/addons/numpy/random/__init__.py", line 99, in <module>
    from .mtrand import *
ImportError: dlopen(/Users/brady/Library/Application Support/Blender/2.80/scripts/addons/numpy/random/mtrand.so, 2): Symbol not found: __PyThreadState_Current
  Referenced from: /Users/brady/Library/Application Support/Blender/2.80/scripts/addons/numpy/random/mtrand.so
  Expected in: dynamic lookup
```

When running python from the /Applications/Blender.app/Contents/Resources/2.80/python/bin/python3.7m I can import numpy without a problem.
```python
Bradys-MacBook-Pro:~ brady$ cd /Applications/Blender.app/Contents/Resources/2.80/python/bin
Bradys-MacBook-Pro:bin brady$ ./python3.7m 
Python 3.7.0 (default, Sep 11 2018, 17:36:20) 
[Clang 9.1.0 (clang-902.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy
>>> 
```

I've tried with the other release candidates but no luck. 

System details: 
![image](https://user-images.githubusercontent.com/36021261/62035966-7222e600-b223-11e9-86fa-f3565b66deb0.png)

Any help would be fantastic - but if I should be submitting it to Blender instead let me know.

**Comment by OmarSquircleArt** [comment]
This seems to be related to dynamic linking in OSX, which I don't know much about. Did you edit your system's dynamic library paths or something like that?

**Comment by Clockmender** [comment]
@BradyAJohnston Redact your serial number - there are bad people out there...

**Comment by BradyAJohnston** [comment]
> @BradyAJohnston Redact your serial number - there are bad people out there...

Thanks for that, it's been a long day.

**Comment by BradyAJohnston** [comment]
> This seems to be related to dynamic linking in OSX, which I don't know much about. Did you edit your system's dynamic library paths or something like that?

I've added some paths trying to get some custom python apps to work, but shouldn't blender be operating on it's own self-contained python?

**Comment by OmarSquircleArt** [comment]
You are right. I don't really know what the issue here, so you should probably wait until a more experienced developer take a look. Maybe the numpy build is corrupt somehow, can you try replacing this numpy build with another one? Maybe downloaded using pip?

**Comment by Clockmender** [comment]
Check you have numpy here:

![Screen Shot 2019-07-29 at 10 35 43](https://user-images.githubusercontent.com/29657804/62037993-c2cb1c80-b1ec-11e9-9863-04bb657fdbc1.png)

(Substitute your user name), If not try opening a terminal and enter command:

`python3.7m -pip install numpy`

See what you get - this will install in the Blender Python directory. You may like to try removing it first, then reinstalling.

Cheers, Clock.

**Comment by BradyAJohnston** [comment]
Okay so I have a fix after tinkering around in site-packages. I went through and manually removed files for numpy wherever I could find it relating to blender. 

```
~/Library/Application\ Support/Blender/2.80/scripts/addons/
/Applications/Blender.app/Contents/Resources/2.80/python/lib/python3.7/site-packages
/Applications/Blender.app/Contents/Resources/2.80/python/bin

```
After I went through and removed numpy files, including the `numpy-1.17.0.dist-info` from `site-packages`, I was able to do a fresh install `./python3.7m -m pip install numpy` using Blender's packaged python.

Now installs fine with 2.1.4, thanks for the quick responses everyone.
